src/app/docs/flyout/flyout-docs.component.html

```diff
@@ -65,4 +66,395 @@
+          The flyout covers the omnibar, but the help panel remains visible in the upper right corner of the screen. The help panel should recognize that the flyout is open and provide appropriate guidance based on the context.
+        </p>
+
+        <table class="sky-docs-table">
+          <tr>
+            <td class="sky-docs-table-cell">
+              <sky-docs-thumbnail
+                imageSource="~/assets/img/guidelines/flyout/ixn-help-1.png"
+                showBorder="true"
+              >
+              </sky-docs-thumbnail>
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-thumbnail
+                imageSource="~/assets/img/guidelines/flyout/ixn-help-2.png"
+                showBorder="true"
+              >
+              </sky-docs-thumbnail>
+            </td>
+          </tr>
+        </table>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Modal dialog behavior"
+      >
+        <p class="sky-docs-text-block">
+          When users perform actions in the flyout to launch modal dialogs, the modal dialogs should cover the entire page.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/flyout/ixn-modal.png"
+          showBorder="true"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Options"
+    >
+      <sky-docs-design-guideline
+        heading="Options"
+      >
+        <p class="sky-docs-text-block">
+          When a flyout opens records in a list, you can include buttons in the toolbar to open the next record or previous record. For example, if users open a flyout on a constituent record to view the first gift in a list of gifts, the next button opens the second gift in the list. When you include the next and previous buttons in the flyout toolbar, a visual indicator highlights the currently selected row on the page.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/flyout/opt-arrows.png"
+          showBorder="true"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="List toolbar"
+      >
+        <p class="sky-docs-text-block">
+          When a flyout displays a list, you can include an option in the toolbar to leave the "preview" list and create a new list for further manipulation. The create list button navigates to a new page that contains the new list.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/flyout/opt-list.png"
+          showBorder="true"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Mobile behavior"
```

Comment:
```suggestion
        heading="Responsiveness"
```
Also, I think we should move this section to be a subsection in the Behavior and states section. I searched for existing Design guidelines with Responsiveness sections, and all the ones I found were in the Behavior and states section, not the Options section.

Reply:
Just following up to see if you decided against the second suggestion. If you think this particular *Responsiveness* section should be under *Options*, that works. But all the other *Responsiveness* sections I found were under *Behavior and states*.

Reply:
I made the heading change but overlooked moving the section. That's done now.